runbotics-orchestrator/src/main/java/com/runbotics/repository/UserBotRepository.java

```diff
@@ -0,0 +1,33 @@
+package com.runbotics.repository;
+
+import com.runbotics.domain.UserBot;
+import com.runbotics.domain.UserBotPK;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Spring Data JPA repository for the {@link UserBot} entity.
+ */
+@Repository
+public interface UserBotRepository extends JpaRepository<UserBot, UserBotPK> {
+
+    Optional<UserBot> findByIdUserIdAndIdBotId(Long userId, Long botId);
+
+    @Query(
+        value = "SELECT * FROM jhi_user ju INNER JOIN jhi_user_bot jub ON ju.id = jub.user_id WHERE jub.bot_id = ?1",
```

Comment:
So maybe it's better change it to JPA query instead of nativeQuery? 
It will use class column name instead of database, and it's easly to change database/tests (I know we don't have tests :D )